packages/contracts-bedrock/package.json

```diff
@@ -80,8 +80,8 @@
     "forge-std": "https://github.com/foundry-rs/forge-std.git#46264e9788017fc74f9f58b7efa0bc6e1df6d410",
     "glob": "^7.1.6",
     "hardhat-deploy": "^0.11.4",
-    "@safe-global/safe-contracts": "https://github.com/safe-global/safe-contracts.git#v1.3.0-libs.0",
-    "solady": "https://github.com/Vectorized/solady.git#v0.0.85",
+    "@safe-global/safe-contracts": "https://github.com/safe-global/safe-contracts.git#cb4b2b19b3e336b8defd3b8c9e0e6a2ae130598c",
```

Comment:
We may need to fully remove the `.git` suffix.

Reply:
it was previously working on other packages (ie. `forge-std` above), but I'll give it a shot.